sf_stats.Api/sf_stats.Domain/Dtos/GameDto.cs

```diff
@@ -0,0 +1,32 @@
+﻿using sf_stats.Domain.Entities;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace sf_stats.Domain.Dtos
+{
+    public class GameDto
+    {
+        public int Id { get; set; }
+
+        [ForeignKey("Season")]
```

Comment:
I think we are mixing Data Annotations with Fluent API here.

[Has Foreign Key example](https://www.learnentityframeworkcore.com/configuration/fluent-api/hasforeignkey-method)

[EF Navigation properties example](https://docs.microsoft.com/en-us/ef/core/modeling/relationships?tabs=fluent-api%2Cfluent-api-simple-key%2Csimple-key)

Reply:
probably. I was just trying some things I saw online until it worked. Now that I see how it all goes together I'll try and clean it up